app/src/main/java/com/infomaniak/mail/ui/main/thread/MessageWebViewClient.kt

```diff
@@ -101,7 +103,22 @@ class MessageWebViewClient(
 
     override fun onPageFinished(webView: WebView, url: String?) {
         runCatchingRealm {
-            webView.loadUrl("javascript:removeAllProperties(); normalizeMessageWidth(${webView.width.toDp()}, '$messageUid')")
+            val widthInDp = webView.width.toDp()
+            if (widthInDp <= 0) {
+                Sentry.withScope { scope ->
+                    scope.level = SentryLevel.WARNING
+                    scope.setExtra("width", webView.width.toString())
+                    scope.setExtra("measuredWidth", webView.measuredWidth.toString())
+                    scope.setExtra("height", webView.height.toString())
+                    scope.setExtra("measuredHeight", webView.measuredHeight.toString())
+                    scope.setTag("visibility", webView.visibility.toString())
+                    scope.setTag("messageUid", messageUid)
```

Comment:
I'm not sure this is a useful info for us, the only use I thought of is seeing if for a given user, it's always the same message that bug
Is that what you want ?

Reply:
Kind of, maybe also seeing if a given message will always trigger the issue, this kind of thing. Not that this tells us straight forward the answer, but could give us a hint